### PR TITLE
Fix issue with MQC not displaying post-ar trimming FastQC results

### LINKED ISSUE #1073: MultiQC doesn't display post artrimming FASTQC results
Because forgot to include those suffixes in the multiqc path filters

Thanks  to  @kieren-j-mitchell  for reporitng!

### LINKED ISSUE #1079: Add lane merging output directory to output documentation.
Currently the output directory `lanemerging` is not documented in the output files section of the output docs.

### PR DESCRIPTION
**Pull request by jfy133**
Closes #1073 
Closes #1079 
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/eager/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +23,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### `Fixed`
 
-- [#1048](https://github.com/nf-core/eager/issues/1048) `--vcf2genome_outfile` parameter now gets prefixed by the sample_name and suffixed with `.fasta` (i.e. `<sample_name>_<vcf2genome_outfile>.fasta`). This ensures we avoid overwriting the output fasta of one sample with that of another when the option is provided. (♥ Thanks to @MeriamOs for reporting.)
+- [#1048](https://github.com/nf-core/eager/issues/1048) `--vcf2genome_outfile` parameter now gets prefixed by the sample_name and suffixed with `.fasta` (i.e. `<sample_name>\_<vcf2genome_outfile>.fasta`). This ensures we avoid overwriting the output fasta of one sample with that of another when the option is provided. (♥ Thanks to @MeriamOs for reporting.)
```
Comment (jfy133) [inline CHANGELOG.md:26]:
```suggestion
- [#1048](https://github.com/nf-core/eager/issues/1048) `--vcf2genome_outfile` parameter now gets prefixed by the sample_name and suffixed with `.fasta` (i.e. `<sample_name>_<vcf2genome_outfile>.fasta`). This ensures we avoid overwriting the output fasta of one sample with that of another when the option is provided. (♥ Thanks to @MeriamOs for reporting.)
```

**Review by TCLamnidis — APPROVED** [review]

**Comment by jfy133** [comment]
@TCLamnidis can you do release procedure?

**Comment by TCLamnidis** [comment]
@jfy133 happy with the wording of the lanemerge output docs?

File: docs/output.md [hunk 2]
```diff
@@ -680,6 +680,7 @@ Each module has it's own output directory which sit alongside the `MultiQC/` dir
   * When masking of the reference is requested prior to running pmdtools, an additional directory `reference_genome/masked_genome` will be found here, containing the masked reference.
 * `fastqc/`: this contains the original per-FASTQ FastQC reports that are summarised with MultiQC. These occur in both `html` (the report) and `.zip` format (raw data). The `after_clipping` folder contains the same but for after AdapterRemoval.
 * `adapterremoval/`: this contains the log files (ending with `.settings`) with raw trimming (and merging) statistics after AdapterRemoval. In the `output` sub-directory, are the output trimmed (and merged) `fastq` files. These you can use for downstream applications such as taxonomic binning for metagenomic studies.
+* `lanemerging/`: this contains adapter-trimmed and merged (i.e. collapsed) FASTQ files that were merged across lanes, where applicable. These files are used for downstream applications such as taxonomic binning for metagenomic studies.
```
Comment (jfy133) [inline docs/output.md:683]:
```suggestion
* `lanemerging/`: this contains adapter-trimmed and merged (i.e. collapsed) FASTQ files that were merged across lanes, where applicable. These files are used for downstream applications such as taxonomic binning for metagenomic studies, and the reads that go into mapping.
```

**Review by jfy133 — COMMENTED** [review]
Yes, extra just to clarify given the original context. 

Don't forget to put yourself as the person making the fix

File: docs/output.md (hunk 2 above)
Comment (TCLamnidis) [reply docs/output.md:683]: I tweaked the wording a bit more

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,20 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.5.2] - 2024-06-07
+
+### `Added`
+
+- [#1079](https://github.com/nf-core/eager/issues/1079) - Added the `lanemerging` output directory in the output documentation (♥ to @TessaZei for reporting).
```
Comment (TCLamnidis) [inline CHANGELOG.md:10]:
```suggestion
- [#1079](https://github.com/nf-core/eager/issues/1079) - Added the `lanemerging` output directory in the output documentation (♥ to @TessaZei for reporting, and @TCLamnidis for fixing).
```

**Comment by TCLamnidis** [comment]
CIs passed. Linting failing due to DSL2 template. Will merge 🙏

**Pull request merged**